cmd/kubernetes-static/data/1_18/controlplane/controller-manager/metrics

```diff
@@ -1,2448 +0,0 @@
-# HELP ClusterRoleAggregator_adds (Deprecated) Total number of adds handled by workqueue: ClusterRoleAggregator
-# TYPE ClusterRoleAggregator_adds counter
-ClusterRoleAggregator_adds 502
```

Comment:
Something weird happened. It shows that 1.18 was deleted and tha 1.15 was moved to be called 1.18 in an other folder.

I fear the files for all the version were the same and git got confused 😓

Reply:
I thing git went bonkers. From what I can tell, I just moved the things and then deleted the old ones.

But in any case these will be deleted once we regenerate the static files, which will hapen in the next PR in this line.